src/Database/Schema/SqlserverSchema.php

```diff
@@ -348,11 +348,12 @@ public function columnSql(TableSchema $schema, $name)
             $out .= $typeMap[$data['type']];
         }
 
-        if ($data['type'] === 'integer' || $data['type'] === 'biginteger') {
-            if ([$name] === $schema->primaryKey() || $data['autoIncrement'] === true) {
-                unset($data['null'], $data['default']);
-                $out .= ' IDENTITY(1, 1)';
-            }
+        if (
```

Comment:
Ahh, forgot to cherry-pick the commit that fixed that. Pulled it into the PR now.